http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfigurationBuilder.java

```diff
@@ -176,6 +177,16 @@ public HttpClient call() {
         };
     }
 
+    public static OidcClientConfiguration buildWithoutUnsupportedAttributes(String unsupportedAttributesParam, InputStream is) {
```

Comment:
Maybe this method could be updated to `build(InputStream is, String unsupportedOidcAttributes)`?

Reply:
fixed!